### ISSUE TITLE
Support for Managing GuardDuty Organization Configuration

### ISSUE THREAD

**Issue by bflad** [terraform-providers/terraform-provider-aws#13006]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

AWS recently announced Organizations support in GuardDuty. One part of that functionality is the ability to configure the organization.

### New Resource

* aws_guardduty_organization_configuration

### Potential Terraform Configuration

```hcl
resource "aws_guardduty_organization_configuration" "example" {
  auto_enable = true
  detector_id = aws_guardduty_detector.master.id
}
```

### References

- [Announcement](https://aws.amazon.com/about-aws/whats-new/2020/04/amazon-guardduty-simplifies-multi-account-threat-detection-with-support-for-aws-organizations/)
- [User Guide](https://docs.aws.amazon.com/guardduty/latest/ug/guardduty_organizations.html)
- [`DescribeOrganizationConfiguration`](https://docs.aws.amazon.com/guardduty/latest/APIReference/API_DescribeOrganizationConfiguration.html)
- [`UpdateOrganizationConfiguration`](https://docs.aws.amazon.com/guardduty/latest/APIReference/API_UpdateOrganizationConfiguration.html)

**Comment by bflad** [comment]
Support submitted: https://github.com/terraform-providers/terraform-provider-aws/pull/13034

**Comment by bflad** [comment]
Support for this new resource has been merged and will release with version 2.60.0 of the Terraform AWS Provider, likely later today. 👍
